wakuv2/waku.go

```diff
@@ -661,7 +660,7 @@ func (w *Waku) subscribeToPubsubTopicWithWakuRelay(topic string, pubkey *ecdsa.P
 				}
 				return
 			case env := <-sub[0].Ch:
-				err := w.OnNewEnvelopes(env, common.RelayedMessageType, false)
+				err := w.OnNewEnvelopes(env, common.MissingMessageType, false)
```

Comment:
Wondering why it's MissingMessageType here, isn't this channel for relayed messages?

Reply:
Oops. I modified the wrong line. Fixed on last commit. Thank you